### PR TITLE
Add support for specifying SSL certificate & key

### PR DESCRIPTION
**Pull request by jkrehm**
Adds CLI options to specify `--cert` and `--key` alongside `--https`. If they are set and the files found, they are used. If they are not set but `--https` is, then the `generateCertificate` logic is called. If `--https` is not specified, then an insecure server will start.

### CODE REVIEW

**Comment by jkrehm** [comment]
Anything I can do to improve this PR and increase its chances of making it into the project?

File: src/utils/getCertificate.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+function getCertificate(options) {
+  const key = fs.existsSync(options.key) && fs.readFileSync(options.key);
```
Comment (DeMoorJasper) [inline src/utils/getCertificate.js]: Perhaps use async reads here?

Comment (jkrehm) [reply src/utils/getCertificate.js]: I can add that. I patterned its behavior after `generateCertificate` which uses sync calls. I'll update the PR to use `utils/fs.js` and `await`.

**Comment by jkrehm** [comment]
Changes made to load the files asynchronously.

**Comment by devongovett** [comment]
Updated to make the `--https` option not necessary when using `--cert` and --key` - we can infer https from those keys.

**Comment by jkrehm** [comment]
Thanks!

**Pull request merged**